### DISCUSSION TITLE
Found Inkline Last Night Decided to Create a Quick Start Project

### DISCUSSION THREAD

**Discussion by MRSessions** [inkline/inkline#340] [Showcase]
Hey all,

As the title says, I found Inkline last night in an effort to find a new UI framework for Vue to play with on a couple of small personal project ideas I have.

I decided to create a quick startup project for myself to use for these and figured I would share with the community.

It's super basic but it's meant to get you up and running fast with a header, footer, and main content area.

Enjoy,
Matt

Repo: https://github.com/MRSessions/Inkline-Starter-Template
Demo: https://green-mushroom-01320b310.2.azurestaticapps.net/

**Comment by alexgrozav** [comment]
This is awesome! Thank you so much for the initiative. 😍
